source/HealthNode.cpp

```diff
@@ -0,0 +1,53 @@
+#include "HealthNode.h"
+
+#include <cugl/2d/CUAnimationNode.h>
+
+using namespace cugl;
+
+/** The diameter of the ship. Also the x coordinate of the center of the ship */
+constexpr unsigned int DIAMETER = 1280;
+
+/** The radius of the ship. */
+constexpr unsigned int RADIUS = 550;
+
+const Vec2 pos0 = Vec2(DIAMETER + RADIUS * sin(0), DIAMETER / 2.0f - (RADIUS - 345) * cos(0));
+
+const Vec2 pos1 = Vec2(DIAMETER + 150 + RADIUS * sin(0), DIAMETER / 2.0f - (RADIUS - 410) * cos(0));
+
+const Vec2 pos2 = Vec2(DIAMETER + 202 + RADIUS * sin(0), DIAMETER / 2.0f - (RADIUS - 550) * cos(0));
+
+const Vec2 pos3 = Vec2(DIAMETER + 145 + RADIUS * sin(0), DIAMETER / 2.0f + (RADIUS - 410) * cos(0));
+
+const Vec2 pos4 = Vec2(DIAMETER + RADIUS * sin(0), DIAMETER / 2.0f + (RADIUS - 345) * cos(0));
+
+const Vec2 pos5 = Vec2(DIAMETER - 145 + RADIUS * sin(0), DIAMETER / 2.0f + (RADIUS - 410) * cos(0));
+
+const Vec2 pos6 = Vec2(DIAMETER - 202 + RADIUS * sin(0), DIAMETER / 2.0f - (RADIUS - 550) * cos(0));
+
+const Vec2 pos7 = Vec2(DIAMETER - 150 + RADIUS * sin(0), DIAMETER / 2.0f - (RADIUS - 410) * cos(0));
+
+void HealthNode::draw(const std::shared_ptr<cugl::SpriteBatch>& batch, const Mat4& transform,
+					  Color4 tint) {
+	if (section == 0) {
+		setPosition(pos0);
+	} else if (section == 1) {
+		setPosition(pos1);
+	} else if (section == 2) {
+		setPosition(pos2);
+	} else if (section == 3) {
+		setPosition(pos3);
+	} else if (section == 4) {
+		setPosition(pos4);
+	} else if (section == 5) {
+		setPosition(pos5);
+	} else if (section == 6) {
+		setPosition(pos6);
+	} else {
+		setPosition(pos7);
+	}
```

Comment:
Feels like this should probably be a switch statement. Also is there no closed form solution for computing this? We just have to have 7 hard-coded constants?

Reply:
i think there is a better way to compute it but in terms of look it didn't look as good when having some equation compute it. I want to update this in the next sprint once we have ship scaling, so this isn't permanent